krmfn/runner.go

```diff
@@ -0,0 +1,68 @@
+package krmfn
+
+import (
+	"fmt"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+var runnerErr error
+
+type Runner struct {
```

Comment:
Why can't `executeFn` struct implement the interface directly instead of creating a new struct Runner?

Reply:
I was thinking that it would be cleaner and also if we use `executeFn` struct, then a user need not build the object builder as `executeFn` will be exported and they can directly run `executeFn.Execute` instead of `executeFn.build` -> `executeFn.Execute`